### PR TITLE
845291-Update MAUI ImageEditor Background API content.

### PR DESCRIPTION
**Pull request by Rishvanth-Kumaresan**
Hi @saiganesh-sakthivel ,

I have added the content for the Background API. Kindly review the content and inform me if any changes required.

Regards,
Rishvanth Kumaresan

### CODE REVIEW

File: MAUI/ImageEditor/getting-started.md [hunk 1]
```diff
@@ -307,4 +307,17 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## Set the Background to the ImageEditor
+The background of the ImageEditor can be customized by setting the Background property of the SfImageEditor.
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:313]: Highlight API Background

File: MAUI/ImageEditor/getting-started.md [hunk 2]
```diff
@@ -307,4 +307,17 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## Set the Background to the ImageEditor
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:312]: Change the Image Editor Background

File: MAUI/ImageEditor/getting-started.md (hunk 1 above)
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/getting-started.md:313]: The background of the Image Editor

Comment (Rishvanth-Kumaresan) [reply MAUI/ImageEditor/getting-started.md]: Highlighted Background API.

File: MAUI/ImageEditor/getting-started.md (hunk 2 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/ImageEditor/getting-started.md]: Updated topic.

File: MAUI/ImageEditor/getting-started.md (hunk 1 above)
Comment (Rishvanth-Kumaresan) [reply MAUI/ImageEditor/getting-started.md]: Updated the content.

**Review by Chrispineagunja — COMMENTED** [review]
This content is fine.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**